modules/gateway-task/variables.tf

```diff
@@ -120,6 +120,13 @@ variable "consul_https_ca_cert_arn" {
   default     = ""
 }
 
+# TODO: Check how to deal with this one.
```

Comment:
Not clear to me how to proceed with `consul_https_ca_cert_arn`. The usage seems to be at task level. Need some guidance here.